source/ingress.go

```diff
@@ -152,7 +160,11 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 			continue
 		}
 
-		endpoints = addEndpointsFromTargetAnnotation(ing, rule.Host, endpoints)
+		endpoints = getEndpointsFromTargetAnnotation(ing, rule.Host)
```

Comment:
No, I think that's accidental.  I'll take a look at fixing that and get this updated if I'm correct.